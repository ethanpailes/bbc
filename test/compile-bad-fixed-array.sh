#!/bin/sh

BBCC=$(stack exec which bbc)

${BBCC} -o fixed-array-type-error fixed-array-type-error.bb

exit $?
