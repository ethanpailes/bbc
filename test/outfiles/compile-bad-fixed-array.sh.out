bbc: FixedArray (Tycon "inner") 256 is not a valid type for used in a fixed length array
